backend/src/libs/types/service.type.ts

```diff
@@ -2,7 +2,7 @@ type Service<T = unknown> = {
   find(): Promise<T>;
   findAll(): Promise<{
     items: T[];
-  }>;
+  } | null>;
```

Comment:
return empty array instead
```suggestion
  }>;
```